README.md

```diff
@@ -6,7 +6,7 @@ Pulls multiple podcast feeds (RSS) and republishes as a common feed, properly so
 
 - Copy `docker-compose.yml` and adjust exposed port if needed
 - Create `etc/fm.yml` (sample provided in `_example`)
-- Start container with `docker-compose up -d`
+- Start container with `docker-compose up -d feed-master`
```

Comment:
this need a little bit of explantion about the local tg api service running
I would suggest adding a separate secition to this readme just about tg 50 limitation and the aproach with the local tg server.